content/docs/ops/security-ir-checklist.md

```diff
@@ -85,3 +85,7 @@ If a cloud.gov team member or automated scanning system detects unauthorized acc
   - VPC Flow Logs: from the Amazon AWS console select `Services -> VPC -> VPC ID -> Flow Logs`
   - Live capture: from the Isolation Segment Diego Cell run `tcpdump -i $INTERFACE -s 65535 -w /tmp/incident-$(date +%s).pcap`
   - Customer: the customer has control of all systems on the customer side of the VPN, so the customer needs to capture that traffic
+
+### Github Secret Leak
+
+If we've leaked secrets in Github, _once the Incident Response team has told us to remove the data from Github_, follow [these instructions to remove the secret from Git history](https://help.github.com/en/github/authenticating-to-github/removing-sensitive-data-from-a-repository), and then file a [support ticket to have the files removed from their cache](https://support.github.com/contact).
```

Comment:
Did they confirm this? It seems to fall outside the usual "don't tamper with evidence" dictum.

Reply:
I think that's covered by `_once the Incident Response team has told us to remove the data from Github_`